test-three-nathanpeter/menu/src/data/menudata.js

```diff
@@ -0,0 +1,63 @@
+export const menu = {
+    "name": "menu",
+    "children": [
+      {
+        "name": "Account",
+        "firstchild": [
```

Comment:
curious as to why you renamed these children keys

Reply:
just made more sense for me to be able to map through them conditionally so I can access it directly